website/src/pages/download.mdx

```diff
@@ -0,0 +1,40 @@
+import { customFields } from '../../docusaurus.config.json'
+
+export const Link = () => {
+  const curr = `https://github.com/GMOD/jbrowse-components/releases/tag/${customFields.currentVersion}`
+  return <a href={curr}>{curr}</a>
+}
+
+# JBrowse 2 download
+
+To get started with JBrowse 2, we recommend using the "jbrowse CLI"
```

Comment:
What do you think about updating this to say "JBrowse 2 web" so people can get used to the distinction between the web and desktop products

Reply:
Currently there are many places in our website that refer to "JBrowse Web". I am not super enthused about this styling of our product name though. I would almost prefer to leave as is, and add a separate section about desktop users because that should require much less explanation

Reply:
trying to keep with the style and changed most mentions to "JBrowse Web" now